config/crd/bases/postgres-operator.crunchydata.com_postgresclusters.yaml

```diff
@@ -3101,10 +3103,14 @@ spec:
                                       description: volumeName is the binding reference
                                         to the PersistentVolume backing this claim.
                                       type: string
-                                  required:
-                                  - accessModes
-                                  - resources
                                   type: object
+                                  x-kubernetes-validations:
+                                  - message: missing accessModes
+                                    rule: has(self.accessModes) && size(self.accessModes)
+                                      > 0
+                                  - message: missing storage request
+                                    rule: has(self.resources) && has(self.resources.requests)
+                                      && has(self.resources.requests.storage)
```

Comment:
🤔 This affects the API reference documentation. The **required** column changes from yes to no.